go/pkg/uploadinfo/entry.go

```diff
@@ -59,3 +65,14 @@ func EntryFromFile(dg digest.Digest, path string) *Entry {
 		ueType: uePath,
 	}
 }
+
+// EntryFromVirtualFile creates an entry from a file not on disk.
+// The digest is expected to exist in the CAS.
+func EntryFromVirtualFile(dg digest.Digest, path string) *Entry {
+	return &Entry{
+		Digest:      dg,
+		Path:        path,
+		ueType:      ueBlob,
```

Comment:
```suggestion
		ueType:      ueFile,
```

Reply:
Don't we want to use blobs for virtual files?

Reply:
My suggestion was from a consistency perspective. Since those are not meant to be uploaded, it doesn't matter what the type is.